Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,21 @@
 import React from 'react';
+import Button from './Button';
 import './Button.css';
+
+let buttonStyle = {
+  width: '75%',
+  height: '100px',
+  border: '1px solid grey',
+  backgroundColor: 'white',
+  fontSize: '3.5rem',
+  fontWeight: 200
+};
+
+export default (props) => {
+  if (props.text === '0') {
```

Comment:
Why using an anonymous function here?
It working fine, but conventionally you call your component in the same way of the module containing it.